src/cryptoadvance/specter/templates/includes/helpers.jinja

```diff
@@ -1,26 +1,189 @@
 <!-- This file contains javascript helper functions that also can use jinja -->
 <script type="text/javascript">
 
-    // Formats an unitLabel (e.g. from a tx output) to a useful unit label
-    function formatUnitLabel(unitLabel, convertToSat){
-        var newLabel = unitLabel;
+
+
+    /*
+        Formats the btc amount as sats. 
+        Args:
+            value (Union[float, str]): (in BTC) Will convert string to float. 
+            enableDigitFormatting (bool, optional): Will group the Satoshis into blocks of 3,
+                e.g. 3 123 456, and color the blocks. Defaults to false.
+        Returns:
+            str: The formatted btc amount as html code.
+        */
+    function internalFormatBtcAmountAsSats(
+        value,
+        enableDigitFormatting=false,
+    ) {
+        const locale = 'en-US';
+        if (isNaN(parseFloat(value))){
+            return parseFloat(value)        
+        }
+        var s = (parseFloat(value) * 1e8).toLocaleString(locale, { 
+            minimumFractionDigits: 0, 
+            maximumFractionDigits: 0 
+        });         
+
+        let thousandsSeparator = Number(1000).toLocaleString(locale).charAt(1);
+        let decimalSeparator = Number(1.1).toLocaleString(locale).charAt(1);
```

Comment:
```suggestion
        let decimalSeparator = '.';
```

Reply:
no, I left if on purpose general, so in the future switching to international is easier.

Reply:
I added comments in [1ac96e6](https://github.com/cryptoadvance/specter-desktop/pull/1901/commits/1ac96e6473ac762c65c65ddb9d98e304715002ea)